src/aqua/decider/mailbox.aqua

```diff
@@ -0,0 +1,51 @@
+module Mailbox declares poll_mailbox, push_remove_worker
+
+export poll_mailbox, push_remove_worker
+
+import Spell from "@fluencelabs/spell/spell_service.aqua"
+import PeerId from "@fluencelabs/aqua-lib/builtin.aqua"
+
+import Console from "../debug.aqua"
+import SpellId, WorkerId from "../types.aqua"
+
+import spell_log from "../fluence/spell.aqua"
+
+use "../fluence/worker.aqua" as Worker
+
+data RemoveWorker:
+    host_id: PeerId
+    worker_id: WorkerId
+
+data DeciderMessage:
+    remove: ?RemoveWorker
+
+service JsonMsg("json"):
+  parse(str: string) -> DeciderMessage
+  stringify(msg: DeciderMessage) -> string
+
+func process_message(msg: DeciderMessage):
+    if msg.remove != nil:
+        cmd = msg.remove!
+        Worker.remove(cmd.host_id, cmd.worker_id)
+
+func poll_mailbox(spell_id: SpellId):
+    Spell spell_id
+
+    mailbox <- Spell.get_mailbox()
+    spell_log(spell_id, ["mailbox", mailbox])
+    if mailbox.success:
+        for msg <- mailbox.messages:
+            try:
+                message <- JsonMsg.parse(msg)
+                process_message(message)
+                spell_log(spell_id, "will pop")
+                popped <- Spell.pop_mailbox()
+                if popped.str != msg:
+                    spell_log(spell_id, ["произошла чудовищная ошибка. expected", msg, "popped", popped])
+
+func push_remove_worker(spell_id: SpellId, host_id: PeerId, worker_id: WorkerId):
```

Comment:
Who is supposed to call this?

Reply:
Someone who wants to remove the worker. But I didn't manage to make it stable, I guess because `getMailbox` function in `spell` service is wrong order. But I'm not sure.

WDYT if we consider this part unstable, and then refactor it to implement `RemoveDeal`?

Reply:
I mean, WHO is this someone?

Mailbox is filled by the node or by some other spells or manually sent requests. The node won't use this code, since it's aqua. Spells won't be importing this code since it's not a lib (and so unstable as it can be). Only the decider developers will be able to call this code. Do you use it?

So, as no one can use it properly (inlc. the lack of permission checking I guess), maybe we should remove `poll_mailbox` stage at all?

Oh, also I see russian symbols above, guess, need to translate it

Reply:
> Only the decider developers will be able to call this code. Do you use it?

Yes, it is for removing workers. But it's not working correctly right now. I suggest to refactor it later to DealRemove, it will come from blockchain.

Reply:
> So, as no one can use it properly (inlc. the lack of permission checking I guess), maybe we should remove poll_mailbox stage at all?

Actually, _anyone_ can send it. That's a draft for coming feature or being able to remove deals, exit deals and so on.